packages/docs/pages/Box/BoxPage.tsx

```diff
@@ -1,74 +1,92 @@
-import { Box, Flex, H0, H1, H2, Text } from '@bigcommerce/big-design';
+import { Box, Flex, H1, Panel, Text } from '@bigcommerce/big-design';
 import React from 'react';
 import styled from 'styled-components';
 
-import { CodePreview } from '../../components';
+import { CodePreview, PageNavigation } from '../../components';
 import { BoxPropTable, DisplayPropTable, MarginPropTable, PaddingPropTable } from '../../PropTables';
 
-const BoxPage = () => (
-  <>
-    <H0>Box</H0>
+const BoxPage = () => {
+  const items = [
+    {
+      id: 'examples',
+      title: 'Examples',
+      render: () => (
```

Comment:
🍹 Some of these render functions feel like they clutter the files a bit and were a little confusing at first, maybe could use a little more component abstraction. Interesting pattern and def makes things easier to reuse 👍

Reply:
Yeah, I think the future of this is to abstract it into separate "views", though we definitely need a little more brainstorming around how to best achieve this.